be/src/olap/cumulative_compaction_time_series_policy.cpp

```diff
@@ -149,6 +156,12 @@ void TimeSeriesCumulativeCompactionPolicy::calculate_cumulative_point(
             break;
         }
 
+        if (!is_delete && rs->version().first != 0 && rs->version().first != rs->version().second &&
```

Comment:
add comment to explain policy

Reply:
Do you mean set cu point to the first empty rowset? Why?

Reply:
When the machine restarts, it will recalculate cumu point, the merged empty version should be the first, it should be merged again.